### PR TITLE
converted spaces to tabs and remove superfluous tabs\spaces

### PR DESCRIPTION
**Pull request by realpack**
As written in https://github.com/wiremod/wire/wiki/Coding-style
"To make sure the code is readable with all tab sizes, we use tabs for indentation and spaces for alignment."
and "Your code shouldn't have trailing whitespace,..."

### CODE REVIEW

**Comment by realpack** [comment]
Later, I can send a request for corrections to "Non-standard language elements like C-style operators (eg. !, &&), C-style comments (//) and continue are banned. They're a custom feature of Garry's Mod Lua and usually unsupported in other environments."

File: lua/entities/gmod_wire_digitalscreen/init.lua [hunk 1]
```diff
@@ -218,7 +218,7 @@ function ENT:WriteCell(Address, value)
 
 	if Address < 1048500 then -- RGB data
 		if self.Memory[Address] == value or
-		   (value == 0 and self.Memory[Address] == nil) then
+			 (value == 0 and self.Memory[Address] == nil) then
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_digitalscreen/init.lua]: Unwanted.

File: lua/entities/gmod_wire_egp/lib/objects/box.lua [hunk 2]
```diff
@@ -28,5 +28,5 @@ function Obj:Contains(point)
 	point = EGP.ScreenSpaceToObjectSpace(self, point)
 	local w, h = self.w / 2, self.h / 2
 	return -w <= point.x and point.x <= w and
-	       -h <= point.y and point.y <= h
+           -h <= point.y and point.y <= h
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_egp/lib/objects/box.lua]: Unwanted (code alignment used space here).

File: lua/entities/gmod_wire_egp/lib/objects/roundedbox.lua [hunk 3]
```diff
@@ -6,29 +6,29 @@ Obj.CanTopLeft = true
...
+			local scx,scy = srx*(sx-(radius*2))/2 , sry*(sy-(radius*2))/2
+			scx,scy = scx*math.cos(math.rad(angle)) - scy*math.sin(math.rad(angle)),
+						scx*math.sin(math.rad(angle)) + scy*math.cos(math.rad(angle))
+			local a,r = math.rad(div*i+(x*90)), radius
+			local dir = {x=math.sin(-(a+math.rad(angle))),y=math.cos(-(a+math.rad(angle)))}
+			local dirUV = {x=math.sin(-a),y=math.cos(-a)}
+			local ru,rv = (radius/sx),(radius/sy)
+			local u,v = 0.5 + (dirUV.x*ru) + (srx/2)*(1-(ru*2)),
+						0.5 + (dirUV.y*rv) + (sry/2)*(1-(rv*2))
+			polys[#polys+1] = {x=xs+scx+(dir.x*r),	y=ys+scy+(dir.y*r) , u=u,v=v}
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_egp/lib/objects/roundedbox.lua]: Unwanted. Why did you replace the 2 spaces here, between arguments?

File: lua/entities/gmod_wire_egp/lib/objects/roundedboxoutline.lua [hunk 4]
```diff
@@ -7,33 +7,33 @@ Obj.CanTopLeft = true
...
+			local scx,scy = srx*(sx-(radius*2))/2 , sry*(sy-(radius*2))/2
+			scx,scy = scx*math.cos(math.rad(angle)) - scy*math.sin(math.rad(angle)),
+						scx*math.sin(math.rad(angle)) + scy*math.cos(math.rad(angle))
+			local a,r = math.rad(div*i+(x*90)), radius
+			local dir = {x=math.sin(-(a+math.rad(angle))),y=math.cos(-(a+math.rad(angle)))}
+			local dirUV = {x=math.sin(-a),y=math.cos(-a)}
+			local ru,rv = (radius/sx),(radius/sy)
+			local u,v = 0.5 + (dirUV.x*ru) + (srx/2)*(1-(ru*2)),
+						0.5 + (dirUV.y*rv) + (sry/2)*(1-(rv*2))
+			polys[#polys+1] = {x=xs+scx+(dir.x*r),	y=ys+scy+(dir.y*r) , u=u,v=v}
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_egp/lib/objects/roundedboxoutline.lua]: Unwanted. Why did you replace the 2 spaces here, between arguments?

File: lua/entities/gmod_wire_expression2/base/ast.lua [hunk 5]
```diff
@@ -7,98 +7,98 @@ AddCSLuaFile()
...
 -- E2Lib.AST.visitChildren(node, function(child)
---     print(string.format("Node has a child of type %s", child[1]))
---     -- replace (sub x x) → 0
---     if child[1] == "sub" and child[3] == child[4] then
---         return { "literal", child[2], 0, "n" }
---     end
+--	 print(string.format("Node has a child of type %s", child[1]))
+--	 -- replace (sub x x) → 0
+--	 if child[1] == "sub" and child[3] == child[4] then
+--		 return { "literal", child[2], 0, "n" }
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_expression2/base/ast.lua]: Failed? There're still spaces.

File: lua/entities/gmod_wire_expression2/base/ast.lua [hunk 6]
```diff
@@ -7,98 +7,98 @@ AddCSLuaFile()
...
+		end
+	end
+	if next(summary) then
+		str = str .. " [" .. table.concat(summary, ", ") .. "]"
+	end
 
-    str = str .. " (" .. tree[2][1] .. ":" .. tree[2][2] .. ")"
-    local childIndentation = indentation .. "|    "
+	str = str .. " (" .. tree[2][1] .. ":" .. tree[2][2] .. ")"
+	local childIndentation = indentation .. "|	"
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_expression2/base/ast.lua]: Completely unwanted (replaced contents of a string literal)!

File: lua/entities/gmod_wire_expression2/base/optimizer.lua [hunk 7]
```diff
@@ -12,170 +12,170 @@ local Optimizer = E2Lib.Optimizer
...
+	return {"literal", instruction[2], value, type}
 end
 
 for _, operator in pairs({ "add", "sub", "mul", "div", "mod", "exp", "eq", "neq", "geq", "leq",
-                           "gth", "lth", "band", "band", "bor", "bxor", "bshl", "bshr" }) do
-    constantPropagation[operator] = function(instruction)
-        if instruction[3][1] ~= "literal" or instruction[4][1] ~= "literal" then return instruction end
-        return evaluateBinary(instruction)
-    end
+							 "gth", "lth", "band", "band", "bor", "bxor", "bshl", "bshr" }) do
```
Comment (CaptainPRICE) [inline lua/entities/gmod_wire_expression2/base/optimizer.lua]: Yet another unwanted...
I will just stop at this point.

**Comment by CaptainPRICE** [comment]
> we use tabs for indentation and spaces for alignment.

**Comment by thegrb93** [comment]
I don't see any existing PR. He converted spaces to tabs, but yes some spaces such as those in strings shouldn't be converted.

**Comment by CaptainPRICE** [comment]
> I don't see any existing PR.

#1253 
But, seems like it was ended.. And there was one more PR iirc.

**Comment by realpack** [comment]
@CaptainPRICE Yes, I opened the entire folder, and then manually redid it. I had a controversial question, at the expense of tabs in rows. I decided that it was necessary. Thanks for the review.

**Comment by realpack** [comment]
I think this pr is ended. Or not?

**Comment by adosikas** [comment]
There are lots of places where you missed indents (mostly for/if with only one line of code), added *massive* indentations when a line is split and a couple places where there are tabs and a single space before the line.
You should definitely go over it at least once manually or alternatively redo it by using one of the lua pretty-printers/autoformatters you can find on the web.

**Comment by TomyLobo** [comment]
14000 lines changed, this is very likely going to break something.
The gain is minimal, if there is any.
There are disadvantages, even if nothing breaks.
I don't know you and I don't trust you to make the right decision in each of those 14000 cases.
Since GitHub tells me you're new to this repository, I'm going to guess the other developers will think similarly.

**Comment by thegrb93** [comment]
You can use ?w=1 to verify only whitespace changes. Then it's just a matter of making sure no strings were changed. Then as long as it compiles, I'd say it's fine. The only real downside is it'll most likely cause conflicts for other PRs, but that's not a huge issue.

**Comment by TomyLobo** [comment]
From what I gather, people have already found whitespace changes he made inside strings.
You wouldn't detect that with ?w=1.
Along with conflicts (which is a one-time problem), we'll introduce another big blocker for the "blame" feature.

If we want this, ideally, they'd post a script with which they do all the changes and we'd review that instead of the 14000 (manual?) changes he's proposing here.

**Comment by thegrb93** [comment]
I know, that's why I marked it work in progress. Not sure I understand your ideal case. Are you saying do it one script at a time?

**Comment by TomyLobo** [comment]
no i'm saying there should be a script that does the changes.

**Comment by thegrb93** [comment]
Regardless of whether a script does it or it's done manually, it'll need to be reviewed to be sure neither failed.

File: lua/entities/gmod_wire_clutch.lua [hunk 8]
```diff
@@ -274,9 +274,9 @@ function ENT:ApplyDupeInfo(ply, ent, info, GetEntByID)
 		Ent2 = GetEntByID(v.Ent2, game.GetWorld())
 
 		if IsValid(Ent1) and
-		   Ent1 ~= Ent2 and
```
Comment (thegrb93) [inline lua/entities/gmod_wire_clutch.lua]: Looks like it missed some spaces here

**Comment by TomyLobo** [comment]
really, is anyone gonna look at all 14000 changes?

**Comment by thegrb93** [comment]
I'd probably only take 10 min. I'll do it once its ready.

**Comment by TomyLobo** [comment]
14000 lines in 10 minutes is 23 (changed!) lines per second. I doubt that :)

**Comment by realpack** [comment]
I will post another commit later.

File: lua/entities/gmod_wire_clutch.lua (hunk 8 above)
Comment (realpack) [reply lua/entities/gmod_wire_clutch.lua]: Corrected in the last commit. Could not point you as a co-author for some reason.

**Comment by realpack** [comment]
I think the PR is done.

File: lua/entities/gmod_wire_consolescreen/cl_init.lua [hunk 9]
```diff
@@ -295,176 +295,176 @@ local specialCharacters = {
...
+							utf8 = string.format("%c%c", 192 + math.floor (c1 / 64), 128 + (c1 % 64))
+						elseif c1 < 65536 then
+							utf8 = string.format("%c%c%c", 224 + math.floor (c1 / 4096), 128 + (math.floor (c1 / 64) % 64), 128 + (c1 % 64))
+						elseif c1 < 2097152 then
+							utf8 = string.format("%c%c%c%c", 240 + math.floor (c1 / 262144), 128 + (math.floor (c1 / 4096) % 64), 128 + (math.floor (c1 / 64) % 64), 128 + (c1 % 64))
+						end
+
+						if specialCharacters[c1] then
+							self:DrawSpecialCharacter(
+							c1, (tx+0.5)*szx, (ty+0.5)*szy, szx, szy,
```
Comment (adosikas) [inline lua/entities/gmod_wire_consolescreen/cl_init.lua]: Indent

File: lua/entities/gmod_wire_consolescreen/cl_init.lua [hunk 10]
```diff
@@ -295,176 +295,176 @@ local specialCharacters = {
...
+		if self.IntTimer < self.Memory1[2043] then
+			if (self.Flash == false) then
+			self.NeedRefresh = true
+			end
+			self.Flash = true
+		end
+
+		if self.IntTimer >= self.Memory1[2043] then
+			if self.Flash == true then
+			self.NeedRefresh = true
```
Comment (adosikas) [inline lua/entities/gmod_wire_consolescreen/cl_init.lua]: Indent

File: lua/entities/gmod_wire_consolescreen/init.lua [hunk 11]
```diff
@@ -5,94 +5,94 @@ include('shared.lua')
...
+	self:SetMoveType(MOVETYPE_VPHYSICS)
+	self:SetSolid(SOLID_VPHYSICS)
+
+	self.Inputs = WireLib.CreateInputs(self, { "CharX", "CharY", "Char (ASCII/Unicode)", "CharParam (RGBrgb; White=999)", "Clk", "Reset" })
+	self.Outputs = WireLib.CreateOutputs(self, { "Memory" })
+
+	self.Memory = {}
+
+	for i = 0, 2047 do
+	self.Memory[i] = 0
```
Comment (adosikas) [inline lua/entities/gmod_wire_consolescreen/init.lua]: Indent

File: lua/entities/gmod_wire_consolescreen/init.lua [hunk 12]
```diff
@@ -5,94 +5,94 @@ include('shared.lua')
...
+
+	self.Cache = GPUCacheManager(self,true)
 end
 
 function ENT:SendPixel()
-  if (self.Memory[2047] ~= 0) && (self.CharX >= 0) && (self.CharX < 30) &&
-                                 (self.CharY >= 0) && (self.CharY < 18) then
-    local pixelno = math.floor(self.CharY)*30+math.floor(self.CharX)
+	if (self.Memory[2047] ~= 0) && (self.CharX >= 0) && (self.CharX < 30) &&
+								(self.CharY >= 0) && (self.CharY < 18) then
```
Comment (adosikas) [inline lua/entities/gmod_wire_consolescreen/init.lua]: Indent

File: lua/entities/gmod_wire_consolescreen/init.lua [hunk 13]
```diff
@@ -5,94 +5,94 @@ include('shared.lua')
...
-  self.Memory[Address] = value
-  self.Cache:Write(Address,value)
-  return true
+	Address = math.floor(Address)
+	if Address < 0 then return false end
+	if Address >= 2048 then return false end
+	if Address < 2000 then -- text/attribute data
+		if self.Memory[Address] == value then return true end
+	else
+		--	self.Memory[Address] = value
```
Comment (adosikas) [inline lua/entities/gmod_wire_consolescreen/init.lua]: Indent

File: lua/entities/gmod_wire_digitalscreen/init.lua (hunk 1 above)
Comment (adosikas) [inline lua/entities/gmod_wire_digitalscreen/init.lua]: Extra space between indent and code

File: lua/entities/gmod_wire_expression2/base/optimizer.lua [hunk 14]
```diff
@@ -12,170 +12,170 @@ local Optimizer = E2Lib.Optimizer
...
-    end
-    return instruction
+	-- (add 0 x) → x
+	if instruction[3][1] == "literal" and instruction[3][3] == 0 then return instruction[4] end
+	-- (add x 0) → x
+	if instruction[4][1] == "literal" and instruction[4][3] == 0 then return instruction[3] end
+	-- (add (neg x) (neg y)) → (neg (add x y))
+	if instruction[3][1] == "neg" and instruction[4][1] == "neg" then
+		return {"neg", instruction[2], {"add", instruction[2], instruction[3][3], instruction[4][3],
+				__instruction = true}}
```
Comment (adosikas) [inline lua/entities/gmod_wire_expression2/base/optimizer.lua]: Indent

File: lua/entities/gmod_wire_expression2/base/optimizer.lua [hunk 15]
```diff
@@ -12,170 +12,170 @@ local Optimizer = E2Lib.Optimizer
...
+		return {"sub", instruction[2], instruction[4][3], instruction[3][3]}
+	end
+	-- (sub x (neg y) → (add x y))
+	if instruction[4][1] == "neg" then
+		return {"add", instruction[2], instruction[3], instruction[4][3]}
+	end
+	-- (sub (neg x) y) → (neg (add x y))
+	if instruction[3][1] == "neg" then
+		return {"neg", instruction[2], {"add", instruction[2], instruction[3][3], instruction[4],
+				__instruction = true }}
```
Comment (adosikas) [inline lua/entities/gmod_wire_expression2/base/optimizer.lua]: Indent

File: lua/entities/gmod_wire_extbus.lua [hunk 16]
```diff
@@ -1,178 +1,178 @@
...
-        end
-        self.DataBytes = self.DataBytes + 1
-        res = true
-      end
-    end
-    return res
-  end
+	Address = math.floor(Address)
+	if (Address >= 0) and (Address < self.ControlDataSize) then
+	-- [0..15] Address bus settings
```
Comment (adosikas) [inline lua/entities/gmod_wire_extbus.lua]: Indent

File: lua/entities/gmod_wire_extbus.lua [hunk 17]
```diff
@@ -1,178 +1,178 @@
...
+			else
+				table.insert(self.ControlData,0)
+			end
+		end
+	elseif Address == 20 then
+		self.PerformRecursiveScan = Value
+	end
+	return true
+	else
+	local res = false
```
Comment (adosikas) [inline lua/entities/gmod_wire_extbus.lua]: Indent

File: lua/entities/gmod_wire_extbus.lua [hunk 18]
```diff
@@ -1,178 +1,178 @@
...
 
 function ENT:TriggerInput(iname, value)
-  for i = 1,8 do
-    if iname == "Memory"..i then
-          self.Memory[i] = self.Inputs["Memory"..i].Src
-    end
-  end
+	for i = 1,8 do
+		if iname == "Memory"..i then
+				self.Memory[i] = self.Inputs["Memory"..i].Src
```
Comment (adosikas) [inline lua/entities/gmod_wire_extbus.lua]: Indent

File: lua/entities/gmod_wire_consolescreen/init.lua (hunk 13 above)
Comment (realpack) [reply lua/entities/gmod_wire_consolescreen/init.lua]: I see no problems.

**Comment by realpack** [comment]
@adosikas Thanks for the review. Yes, I will review the entire diff soon.

**Comment by realpack** [comment]
> I think the PR is done.

Rechecked the entire diff. It took me half an hour.

**Comment by thegrb93** [comment]
Ok, I'll look later

File: lua/entities/gmod_wire_clutch.lua [hunk 19]
```diff
@@ -277,6 +277,7 @@ function ENT:ApplyDupeInfo(ply, ent, info, GetEntByID)
 		   Ent1 ~= Ent2 and
 		   hook.Run( "CanTool", ply, WireLib.dummytrace(Ent1), "ballsocket_adv" ) and
 		   hook.Run( "CanTool", ply, WireLib.dummytrace(Ent2), "ballsocket_adv" ) then
+
```
Comment (adosikas) [inline lua/entities/gmod_wire_clutch.lua]: Newline

File: lua/entities/gmod_wire_egp/lib/objects/roundedboxoutline.lua (hunk 4 above)
Comment (adosikas) [reply lua/entities/gmod_wire_egp/lib/objects/roundedboxoutline.lua]: Still an issue

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 20]
```diff
@@ -12,180 +12,180 @@ surface.CreateFont( "WireGPU_ErrorFont",{
...
+			for k2,v2 in pairs(v) do
+			self.VM.OpcodeTable[k2] = v2
+			end
+		else
+			self.VM[k] = v
+		end
+	end
+
+	self.VM.ErrorText = {}
+	self.VM.ErrorText[2]	= "Program ended unexpectedly"
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Tabs instead of spaces

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 21]
```diff
@@ -12,180 +12,180 @@ surface.CreateFont( "WireGPU_ErrorFont",{
...
+	self.VM.InternalRegister[32] = nil --IF
+	self.VM.InternalRegister[33] = nil --PF
+	self.VM.InternalRegister[34] = nil --EF
+	self.VM.InternalRegister[45] = nil --BusLock
+	self.VM.InternalRegister[46] = nil --IDLE
+	self.VM.InternalRegister[47] = nil --INTR
+	self.VM.InternalRegister[52] = nil --NIDT
+
+	-- Remove some instructions
+	self.VM.OperandCount[16]	= nil --RD
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Tabs instead of spaces

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 22]
```diff
@@ -219,252 +219,252 @@ end
...
-  --
-  -- Misc:
-  --  [64512] - Last register
-  --  [63488]..[64511] - External ports
-
-  self.Memory[65535] = 1
-  self.Memory[65534] = 0
---self.Memory[65533] = 1   (value persists over reset)
+	-- Reset VM
+	self.IP = 0		-- Instruction pointer
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Tabs again

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 23]
```diff
@@ -219,252 +219,252 @@ end
...
+	self.ES = 0
+	self.GS = 0
+	self.FS = 0
+	self.KS = 0
+	self.LS = 0
+
+	-- Extended registers
+	for reg=0,31 do self["R"..reg] = 0 end
+
+	self.ESZ = 32768	-- Stack size register
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Tabs ... again

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 24]
```diff
@@ -219,252 +219,252 @@ end
...
+	self.IPREC = 48	 -- Integer precision (48 - floating point mode, 8, 16, 32, 64 - integer mode)
+	self.VMODE = 2		-- Vector mode (2D, 3D)
+	self.INTR = 0		 -- Handling an interrupt
+	self.BlockStart = 0 -- Start of the block
+	self.BlockSize = 0	-- Size of the block
+
+	-- Reset internal GPU registers
+	--	[131072]..[2097151] - Extended GPU memory (2MB GPU)
+	--	[131072]..[1048576] - Extended GPU memory (1MB GPU)
+	--	[131072]..[524287]	- Extended GPU memory (512K GPU)
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: More tabs

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 25]
```diff
@@ -473,46 +473,46 @@ end
...
+	for k,v in pairs(self.ModelMatrix) do self.AsyncState.ModelMatrix[k] = v end
+	for k,v in pairs(self.ProjectionMatrix) do self.AsyncState.ProjectionMatrix[k] = v end
 end
 
 function VM:SaveAsyncThread()
-  if not self.AsyncState then
-    self.AsyncState = {}
-    self:Reset()
+	if not self.AsyncState then
+	self.AsyncState = {}
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Indent

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 26]
```diff
@@ -550,43 +550,43 @@ end
...
-  self.EntryPoint0 = 0
-  self.EntryPoint1 = self.EntryPoint1 or 0
-  self.EntryPoint2 = 0
-  self.EntryPoint3 = 0
-  self.EntryPoint4 = 0
-  self.EntryPoint5 = 0
-  self.EntryPoint6 = 0
-  self.EntryPoint7 = 0
+	-- Entrypoints to special calls
+	--	0	DRAW	 Called when screen is being drawn
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Tabs

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 27]
```diff
@@ -595,23 +595,23 @@ end
...
-
-  if (self.Memory[65466] ~= 0) or (self.Memory[65465] ~= 1) then
-    local cu,cv = self.Memory[65464],self.Memory[65463]
-    local tu,tv = u-cu,v-cv
-    local angle,scale = self.Memory[65466],self.Memory[65465]
-    du = scale*(tu*math.cos(angle) - tv*math.sin(angle)) + cu + self.Memory[65462]
-    dv = scale*(tv*math.cos(angle) + tu*math.sin(angle)) + cv + self.Memory[65461]
-  end
+	local texturesOnSide = math.floor(512/self.Memory[65521])
+	local textureX = (1/texturesOnSide) *			 (self.Texture % texturesOnSide)
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: *sigh*

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 28]
```diff
@@ -690,119 +690,119 @@ end
...
+		resultVertex.y = resultCoordinate.y
+	elseif self.VertexPipe == 5 then -- 3D matrix
+		local world
+		if not toScreen then
+			-- Transform into world coordinates
+			world = {}
+
+			for i=0,3 do
+			world[i] = self.ModelMatrix[i*4+0] * vertex.x +
+						self.ModelMatrix[i*4+1] * vertex.y +
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Not aligned

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 29]
```diff
@@ -954,339 +954,339 @@ end
...
+					if (self.DistanceCulling == 1) and (not cullVertex) then
+						local Infront = {}
+						local Behind = {}
+
+						local frontCullDistance = self.Memory[65470]
+						local K = -frontCullDistance
+
+						-- Generate list of vertices which go behind the camera
+						if v1.z - K >= 0
+						then Behind [#Behind  + 1] = v1
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Weird formatting, but could still be indended better

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua [hunk 30]
```diff
@@ -1435,906 +1435,906 @@ end
...
+VM.OpcodeTable[210] = function(self)	--DVXPIPE
+	self:Dyn_Emit("VM.VertexPipe = $1")
+end
+VM.OpcodeTable[211] = function(self)	--DCVXPIPE
+	self:Dyn_Emit("VM.CoordinatePipe = $1")
+end
+VM.OpcodeTable[212] = function(self)	--DENABLE
+	self:Dyn_Emit("$L IDX = $1")
+	self:Dyn_Emit("if IDX == 0 then VM.VertexBufEnabled = 1 end")
+	self:Dyn_Emit("if IDX == 1 then VM.VertexBufZSort	 = 1 end")
```
Comment (adosikas) [inline lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: Tabs again

**Comment by thegrb93** [comment]
Please make absolutely sure you didn't change any strings. I'll look once those are fixed.

**Comment by thegrb93** [comment]
Wonder if something like this would work https://github.com/dptole/lua-beautifier

File: lua/entities/gmod_wire_clutch.lua (hunk 19 above)
Comment (realpack) [reply lua/entities/gmod_wire_clutch.lua]: Strange. I did not add it. Not in its local files.

Comment (realpack) [reply lua/entities/gmod_wire_clutch.lua]: I fixed indent.

Comment (realpack) [reply lua/entities/gmod_wire_clutch.lua]: I blundered. My mistake.

File: lua/entities/gmod_wire_gpu/cl_gpuvm.lua (hunk 29 above)
Comment (realpack) [reply lua/entities/gmod_wire_gpu/cl_gpuvm.lua]: In the source file just like that. Such formatting is used here many times.

**Comment by adosikas** [comment]
"Wait for @adosikas to find *all* the places where i messed up and then selectively fix just those" isn't really the type of behavior i was hoping for when commenting here.

I'm out here can only *strongly* recommend against merging this mess in the current form.

**Comment by realpack** [comment]
@adosikas Thank you for everything. Sorry, did not want to exploit you.
I would like to spend more time on this.

**Comment by realpack** [comment]
Damn me if now is not done.

**Comment by adosikas** [comment]
Damn you indeed ...

obvious screenshot

Again I am not sure how you keep "missing" *obvious* stuff like that.

**Comment by realpack** [comment]
@adosikas It didn't cost me to do this.

**Comment by thegrb93** [comment]
@realpack I appreciated the attempt at least. The code is pretty messy. I think a lua styler script would be a better solution.

**Pull request closed without merging**